global_costmap:
  global_frame: map
  robot_base_frame: rotated_base_link
  update_frequency: 5.0
  publish_frequency: 1.0
  static_map: false
  rolling_window: true
  width: 20.0
  height: 20.0
  resolution: 0.05
  track_unknown_space: true
  transform_tolerance: 1.5
  
  plugins:
    - {name: static_layer,   type: "costmap_2d::StaticLayer"}
    - {name: obstacle_layer, type: "costmap_2d::ObstacleLayer"}
    - {name: inflation_layer, type: "costmap_2d::InflationLayer"}

  obstacle_layer:
    observation_sources: laser_scan
    laser_scan:
      sensor_frame: laser       
      data_type: LaserScan
      topic: scan
      marking: true
      clearing: true

  inflation_layer:
    cost_scaling_factor: 1.0         
    inflation_radius: 0.55           
